doc/docs/resources.md

```diff
@@ -0,0 +1,4 @@
+# Additional Resources
+
+- [Blog Post about a lightweight service manager](https://www.redhat.com/en/blog/hirte-renamed-eclipse-bluechi)
```

Comment:
Just wondering, wouldn't it be better to structure link names? Something like:

<Presentation or blog post name>, <presenter(s)>, <Name of the conference or blog>, <date>

Reply:
I was using the format from [systemd](https://systemd.io/) (under `Videos for Users and Administrators` sections). Both is fine for me. What do you think?

Reply:
Hmm, to be frank I don't like that almost every line starts with "Presentation about" or "Blog post about", the name of the presentation/blog should be descriptive/interesting enough ...

Reply:
Changed.
@mwperina PTAL